scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -161,26 +161,28 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
 	          </strong><br><br>
 	        </td>
 	      </tr>
+      </table>
+      <table
 	      <tr>
 	        <td style="width:50%">
 	          {% if case.synopsis %}
 	            <strong>Synopsis:</strong>
 	            {{ case.synopsis|markdown }}
 	          {% endif %}
 	        </td>
-	        <td>
-	          <strong>Associated phenotypes:</strong><br>
-	          {% if case.phenotype_terms %}
-	            <ul>
-	              {% for pheno in case.phenotype_terms %}
-	                <li>
-	                {{pheno.feature}} - (<a href="{{pheno.hpo_link}}" target="_blank">{{pheno.phenotype_id}})</a>
-	                </li>
-	              {% endfor %}
-	            </ul>
-	          {% else %}
-	            No phenotype terms associated.
-	          {% endif %}
+	        <td class="ml-3 d-flex align-items-start flex-column">
```

Comment:
👍

Reply:
Do I win for this layout?

Reply:
Yes, more flex in there!

Reply:
To my defence, I was first, and it looked about the same. 😉